components/lib/VContainer.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="container">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'VContainer',
+}
+</script>
+
+<style lang="postcss" scoped>
+.container {
+  max-width: var(--max-container-width);
+  width: var(--container-width);
+  margin: 0 auto;
```

Comment:
1. Don't need to set both `width` and `max-width`.
2. Use a `rem` value instead of % for widths.

```suggestion
  max-width: 90rem;  /* 1rem = 16px = 1440px */
  margin: 0 auto;
```